models/comments-by-id.model.js

```diff
@@ -0,0 +1,17 @@
+const db = require('../db/connection');
+
+exports.selectCommentsById = (article_id) => {
+    return db.query(`
+    SELECT * FROM comments 
+    WHERE comments.article_id = $1;`, [article_id])
+    .then((result) => {
+        if (!result.rows.length) {
```

Comment:
Consider the difference between an article that doesn't exist vs an article which does exist but has no comments yet